adam_sulewski/app.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+var express = require('express');
+var fs = require('fs');
+var app = express();
+
+var process = function(req, res, next) {
+  var data = '';
+
+  req.on('data', function(chunk) {
+    data += chunk.toString();
+  });
+  req.on('end', function() {
+    req.body = data;
+    next();
+  });
+  req.on('error', function(err) {
+    console.log(err.stack);
+  });
+};
+
+app.post('/file', process, function(req, res) {
```

Comment:
you could generalize this more by making your route '/:file' and then accessing that in req.params.file to name the file according to whatever is submitted.